lldb/source/Plugins/Trace/intel-pt/TraceIntelPTBundleLoader.cpp

```diff
@@ -108,8 +108,8 @@ TraceIntelPTBundleLoader::CreateEmptyProcess(lldb::pid_t pid,
   //    /*listener*/ nullptr, "trace",
   //    /*crash_file*/ nullptr,
   //    /*can_connect*/ false);
-
-  process_sp->SetID(static_cast<lldb::pid_t>(pid));
+  //
+  // process_sp->SetID(static_cast<lldb::pid_t>(pid));
```

Comment:
do you need to do this? Otherwise all processes loaded from JSON will appear with an invalid PID.

Reply:
I reverted this, given your suggestion in the other thread.